.github/workflows/test.yml

```diff
@@ -0,0 +1,26 @@
+name: Test
+
+on:
+  push:
+    branches: [ main ]
+  pull_request:
+    branches: [ main ]
+
+jobs:
+  build:
+    runs-on: ubuntu-latest
+    strategy:
+      fail-fast: false
+      matrix:
+        java: [ '17', '21' ]
+    steps:
+      - uses: actions/checkout@v4
+      - name: Set up JDK ${{ matrix.java }}
+        uses: actions/setup-java@v3
+        with:
+          distribution: 'temurin'
+          java-version: ${{ matrix.java }}
+      - run: ./gradlew jpackage --info
+      - run: xcrun notarytool store-credentials "notarytool-profile" --apple-id "${{ secrets.APPLE_ID }}" --team-id "12345667" --password "${{ secrets.APPLE_ID_PASSWORD }}"
```

Comment:
teamID needs to be one we use for Jabref as well
I can add the rest of the ids later

Reply:
And we need to have the p12 certs